### PR TITLE
Add Ruby 3.2 to CI

### PR DESCRIPTION
**Pull request by petergoldstein**
Runs green on my fork.

### CODE REVIEW

**Review by codealchemy — APPROVED** [review]

File: .github/workflows/ci.yml
Comment (codealchemy) [inline .github/workflows/ci.yml:17]: @petergoldstein can you please update `on` in this file to [include `pull_requests` and `workflow_dispatch`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) as well? I think that's what's keeping CI from running on your PR.

Comment (petergoldstein) [reply .github/workflows/ci.yml:17]: @codealchemy Done, although now it needs approval from a maintainer to run.

**Comment by codealchemy** [comment]
Thanks!

**Pull request merged**
